### PR TITLE
packaging: Follow Fedora packaging guidelines

### PR DESCRIPTION
**Pull request by siteshwar**
Resolves: https://gitlab.cee.redhat.com/covscan/covscan/-/issues/272

### CODE REVIEW

**Comment by siteshwar** [comment]
@frenzymadness I am not familar with python packaging. Would it be possible for you to review this pull request?

**Comment by siteshwar** [comment]
Something has changed outside this pull request that is causing integration tests to fail.

**Comment by siteshwar** [comment]
> Something has changed outside this pull request that is causing integration tests to fail.

https://github.com/release-engineering/kobo/pull/215

**Comment by siteshwar** [comment]
> I've found only one small nitpick. Other changes here look good to me. I can do a full specfile review later, if you want.

Thanks! I will mark this pull request as ready and wait for some more time, so that others can give it a look.

File: osh.spec [hunk 1]
```diff
@@ -5,7 +5,9 @@ Version:        %{version}
 Release:        1%{?dist}
 License:        GPL-3.0-or-later
 Summary:        Static and Dynamic Analysis as a Service
-Source:         %{name}-%{version}.tar.bz2
+URL:            https://github.com/openscanhub/openscanhub/
+# TODO: This should be changed to releases once we start doing releases through GitHub
+Source:         https://github.com/openscanhub/openscanhub/archive/refs/tags/%{name}-%{version}.tar.bz2
```
Comment (siteshwar) [inline osh.spec:10]: @kdudka Shall we create a release on GitHub from latest tag, so that we can use release url here?

**Comment by frenzymadness** [comment]
I see that I already did a review of the diff. Do you want me to review the full specfile?

**Comment by kdudka** [comment]
@frenzymadness I think reviewing the changes is sufficient.  Thank you for taking care of the review!  Nevertheless, you need to click the `Approve` button in GitHub UI (hidden in `Files changed` -> `Review changes`) so that @siteshwar is allowed to merge this.

**Review by frenzymadness — APPROVED** [review]
This is good to be merged when you resolve the conflict.

**Review by kdudka — APPROVED** [review]
@siteshwar This merge request had some conflicts with the current `main` branch so I have rebased the branch for you.  Please double check before merging this!

File: osh.spec (hunk 1 above)
Comment (kdudka) [reply osh.spec]: I did but it did not do what you had expected.  See #135 for details.

**Pull request merged**